src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.doorbell.android" >
-    
+
     <uses-permission android:name="android.permission.INTERNET" />
-    
+
     <uses-sdk
-        android:minSdkVersion="3"
-        android:targetSdkVersion="18" />
+        android:minSdkVersion="4"
```

Comment:
Is there a change that causes the increase from 3 to 4? I'm sure 3 is practically obsolete so far, but just checking!